ns-build/README.md

```diff
@@ -0,0 +1,60 @@
+ns-build
+================
+
+Build a namespace from many viash config files.
+
+## Usage
+
+This action will run `viash ns build`.
+
+We recommend using a Linux or MacOS runner if possible.
+
+### Inputs available
+
+- `query`: - *optional*. Filter which components get selected by
+  component and namespace name. Can be a regex. Example:
+  “^mynamespace/component1\$”.
+- `query_namespace`: - *optional*. Filter which namespaces get selected
+  by namespace name. Can be a regex. Example: “^mynamespace\$”.
+- `query_name`: - *optional*. Filter which components get selected by
+  component name. Can be a regex. Example: “^component1”.
+- `src`: - *optional*. A source directory containing viash config files,
+  possibly structured in a hierarchical folder structure. Default: src/.
+- `platform`: - *optional*. Acts as a regular expression to filter the
```

Comment:
Maybe add options here? Although it is a pain when we add a new platform. Could be a link to the viash documentation for the platforms... But this link might also change...
Or just leave as is 😄

Reply:
Oh you mean the different platforms. Eh. I'd leave it up to the user to know what they're doing, at least for now.